src/Validator/Validate.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+
+namespace App\Validator;
+
+use App\Entity\Painting;
+use App\Enum\RequestType;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\Validator\Constraints\Required;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+class Validate implements ValidateInterface
```

Comment:
We should have PaintingValidator :

for CreatePainting : 

1- use the PaintingRequestMapper to covert 
    request => CreatePaintingRequest
2- in the PaintingValidator  we should have validateCreateRequest(CreatePaintingRequest) method